core/test/utils_test.jl

```diff
@@ -1,9 +1,9 @@
 @testitem "id_index" begin
     using Dictionaries: Indices
 
-    ids = Indices([2, 4, 6])
-    @test Ribasim.id_index(ids, 4) === (true, 2)
-    @test Ribasim.id_index(ids, 5) === (false, 0)
+    ids = Indices([Ribasim.NodeID(2), Ribasim.NodeID(4), Ribasim.NodeID(6)])
```

Comment:
```suggestion
    ids = Indices(Ribasim.NodeID[2, 4, 6])
```

This works with the extra `Base.convert` method suggested.